package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html

```diff
@@ -136,6 +136,15 @@ <h2><%:Overview%></h2>
 								string.format(' (%s)', nodeinfo.software.autoupdater.branch)
 						%></dd>
 					<%- end %>
+					<% if nodeinfo.software.babeld or nodeinfo.software['batman-adv'] then -%>
+						<dt><%:Mesh protocol%></dt>
+						<% if nodeinfo.software.babeld then -%>
+						<dd>babel <%| nodeinfo.software.babeld.version %></dd>
```

Comment:
```suggestion
						<dd>babeld <%| nodeinfo.software.babeld.version %></dd>
```

Reply:
resolved in d2a5b3f9c187ecaeb232bf9b82e574247fc9e7fa

Reply:
fixup in 77a6b9a947759ed6dad826609535f7667ef47265